packages/core/src/sass/app/_common/components/account-switcher.scss

```diff
@@ -268,14 +268,14 @@
     &__logout {
         align-items: center;
         align-self: flex-end;
-        cursor: pointer;
         display: flex;
         justify-content: flex-end;
         padding: 1.6em 1.3em;
 
         &-text {
             @include typeface(--paragraph-left-normal-black, none);
             color: var(--text-prominent);
+            cursor: pointer;
```

Comment:
Is the requirement to only show `pointer` for the text and not the icon? If no, then should we add this line to the `&-icon` selector too?

Reply:
@mahdipourziaei-binary No requirement didn't mentioned, you're right icon should be contained too. Thanks